### ISSUE TITLE
Plot barchart using wifi strength

### ISSUE THREAD

**Issue by victors0991** [PhilJay/MPAndroidChart#4413]
I am doing an app that do a scan list of wifi in a zone. So I have a problem when I plot that information in barchart using mpandroidchart 3 library.

The problem is that as long as the value approaches zero, the signal is better like the image above. https://imgur.com/a/vYT7ry3

But with my app I do not get the same result, the problem is my axies becasue always begin to zero and that I do not want becasue for example the bar that represent to -21 dbm must be taller that bar that respresents to -70 dbm but wuth my app the opposite happens 

https://imgur.com/a/yHUtaE8

I attched part of my code about create barchart

mChart.setDrawGridBackground(false);
    mChart.getDescription().setEnabled(false);
    mChart.setDrawBorders(false);

    mChart.getAxisLeft().setEnabled(true);
    mChart.getAxisRight().setDrawAxisLine(true);
    mChart.getAxisRight().setDrawGridLines(true);
    mChart.getXAxis().setDrawAxisLine(true);
    mChart.getXAxis().setDrawGridLines(true);

    mChart.getXAxis().setPosition(XAxis.XAxisPosition.BOTTOM);

    mChart.getXAxis().setGranularity(1);
    // enable touch gestures
    mChart.setTouchEnabled(true);

    // enable scaling and dragging
    mChart.setDragEnabled(true);
    mChart.setScaleEnabled(true);

    // if disabled, scaling can be done on x- and y-axis separately
    mChart.setPinchZoom(false);

    mChart.getLegend().setWordWrapEnabled(true);

    mBarChart.setDrawGridBackground(false);
    mBarChart.getDescription().setEnabled(false);
    mBarChart.setDrawBorders(false);

    mBarChart.setDrawValueAboveBar(false);
    mBarChart.getAxisLeft().setEnabled(true);
    mBarChart.getAxisRight().setDrawAxisLine(true);
    mBarChart.getAxisRight().setDrawGridLines(true);
    mBarChart.getXAxis().setDrawAxisLine(true);
    mBarChart.getXAxis().setDrawGridLines(true);

    mBarChart.getXAxis().setPosition(XAxis.XAxisPosition.BOTTOM);

    mBarChart.getXAxis().setGranularity(1);
    // enable touch gestures
    mBarChart.setTouchEnabled(true);

    // enable scaling and dragging
    mBarChart.setDragEnabled(true);
    mBarChart.setScaleEnabled(true);

    // if disabled, scaling can be done on x- and y-axis separately
    mBarChart.setPinchZoom(false);

    final XAxis xAxis = mBarChart.getXAxis();
    xAxis.setPosition(XAxis.XAxisPosition.BOTTOM);

    xAxis.setDrawGridLines(false);
    xAxis.setDrawAxisLine(true);

    //xAxis.setLabelRotationAngle(90);

    YAxis leftAxis =mBarChart.getAxisLeft();
    leftAxis.setInverted(true);
    leftAxis.setAxisMinimum(-100);
    leftAxis.setAxisMaximum(-20);
    leftAxis.setLabelCount(5);

    YAxis rightAxis =mBarChart.getAxisRight();
    rightAxis.setInverted(true);

    rightAxis.setDrawTopYLabelEntry(false);
    rightAxis.setInverted(true);
    rightAxis.setAxisMinimum(-100);
    rightAxis.setAxisMaximum(-20);
    rightAxis.setLabelCount(5);

**Comment by ChaoquanTao** [comment]
hello bro, I have also meet a problem like yours. All the data I wanna to show is negative, but I want to let the large one to be higher. So have you solved this problem, could you share your method, thanks..

**Comment by victors0991** [comment]
Yes I did it.  For example you have 2 values -80 and -60, with your code -80 is higher than 60 in bars. Before you add  values to entrybar, you need to do a substraction example -1×(100-60)= -40 that value you add to bar entry now you need to do a Y formatter to tell that this vale (-40) is -60 in Y axis label

**Comment by victors0991** [comment]
I am not in home, let me know if you can do it

**Comment by ChaoquanTao** [comment]
> I am not in home, let me know if you can do it

thank u bro,it help me a lot

**Comment by ChaoquanTao** [comment]
Sorry to bother you again bro, do you know how to hide the value over the barchart entry, cause in your way , the y axis shows -60 but the entry shows -40.
